Source/Turbo/ViewControllers/HotwireTabBarController.swift

```diff
@@ -76,7 +99,7 @@ public struct HotwireTab: Hashable {
 
     public init(title: String,
                 image: UIImage,
-                selectedImage: UIImage?,
+                selectedImage: UIImage? = nil,
```

Comment:
Providing a default value for a non-required parameter cleans up the code a bit.